### DISCUSSION TITLE
camtraptor logo

### DISCUSSION THREAD

**Discussion by damianooldoni** [inbo/camtraptor#283] [Show and tell]
👋 Welcome!

@NeleMullens added some ideas for a camtraptor hex logo. See #270. I think it's not easy to discsuss such topic in a standard issue. So, I thought to move the _discussion_ to a ... GitHub Discussion 😄 

I will upload the two logo options here below as two separate comments. You can vote by adding a 👍 to the comment with the logo you prefer. If you have a better logo in your mind, please add it as a new comment. Of course, feel free to add a comment also in case you want to tell something more related to this topic.

**Comment by damianooldoni** [comment]
![image](https://github.com/inbo/camtraptor/assets/33662631/f64e297f-809e-45f0-b1c0-7209c0a49cac)

**Reply by peterdesmet** [reply]
I very much like this logo and prefer it over the second one. It is playful and (at least to me) depicts an animal taking a selfie, something quite common with camera traps. @NeleMullens would you be keen (I would totally understand if you're not) to try a logo of a raptor peaking in from the bottom left or right of the logo, so it looks more like a "camera trap selfie" which are often not framed nicely: https://www.google.com/search?q=camera+trap+selfie&tbm=isch (examples [here](https://ciorg.imgix.net/images/default-source/default-album/ci_92267314.jpg?sfvrsn=880ec9a5_5?&auto=compress&auto=format&w=800) and [here](https://npr.brightspotcdn.com/dims4/default/52b5192/2147483647/strip/true/crop/900x554+0+277/resize/880x542!/quality/90/?url=http%3A%2F%2Fnpr-brightspot.s3.amazonaws.com%2Flegacy%2Fsites%2Fwpr%2Ffiles%2F201608%2Fcandid_creatures_book_cover.jpeg))?

**Reply by NeleMullens** [reply]
Thank you both for the positive feedback! @peterdesmet I could try to make another logo as you suggested, but I am currently working on something else as well, so I can’t promise when I will finish it. Also, I didn’t make the proportions exactly to the hex-stickers size/proportion standard, so whichever logo gets chosen for the final one, I’ll have to make some slight adjustments (https://github.com/terinjokes/StickersStandard).

**Reply by PietrH** [reply]
The standard in short:

> A hexagon sticker
must be represented by a regular hexagon where the largest diagonals should measure 5.08 centimeters, and which must remain within 1 millimeter tolerance. The sticker must be oriented with a vertex positioned at the top. 

https://sticker.how/

**Comment by damianooldoni** [comment]
![image](https://github.com/inbo/camtraptor/assets/33662631/dfe4f271-c50e-4d91-9584-c2cee45b6553)
